pilot/pkg/networking/plugin/mixer/mixer.go

```diff
@@ -186,6 +186,8 @@ func buildTransport(mesh *meshconfig.MeshConfig, uid attribute) *mccpb.Transport
 		ReportCluster: model.BuildSubsetKey(model.TrafficDirectionOutbound, "", model.Hostname(telemetry), port),
 		// internal telemetry forwarding
 		AttributesForMixerProxy: &mpb.Attributes{Attributes: attributes{"source.uid": uid}},
+		// TODO(yangminzhu): Make this configurable in mesh config.
+		NetworkFailPolicy: &mccpb.NetworkFailPolicy{Policy: mccpb.FAIL_CLOSE},
```

Comment:
@geeknoid we really _should_ update this option to be `FAIL_DENY` or `DENY_ON_FAIL` (and `FAIL_ALLOW` or `ALLOW_ON_FAIL`) in the api.

Reply:
Agree, please follow up with an option to make it fail-open.

Reply:
Sure, will do so in following PRs.